MediaRecorder.html

```diff
@@ -469,6 +498,22 @@
         </dl>
       </section>
     </div>
+    <section>
+      <h2>Data handling</h2>
+        <p><dfn data-dfn-type="dfn" data-lt="fires a dataavailable event">
+        Firing a dataavailable event </dfn> with a <a
+        href="https://www.w3.org/TR/FileAPI/#blob"><code>Blob</code></a>
+        <var>blob</var> means <a
+        href="https://dom.spec.whatwg.org/#concept-event-fire">firing</a>
+        an event named <code>dataavailable</code> at the current
```

Comment:
I'd make the target an input to this algorithm for consistency with DOM's "fire an event" so that the call sites look pretty much the same.